src/components/FeatureViews/Accounts/ProfileHeading.js

```diff
@@ -231,6 +253,13 @@ class ProfileHeading extends Component {
                         {this.props.isAdmin && this.renderEditButton()}
                     </Grid>
                 </Grid>
+                <Grid>
+                    <Button onClick={this.handleClick()}>
```

Comment:
The onClick event handler's default action is to [automatically trigger this handler](https://javascript.info/default-browser-action) whether or not you've actually clicked. Solve this issue by looking at my comment above.

Reply:
There should not be parentheses here - otherwise the function is called every render, instead of onClick